### PR TITLE
Template for stops too close

### PR DESCRIPTION
**Pull request by KathleenQ**
Test is included.
The template looks like:
Screen Shot 2021-02-11 at 2 03 06 pm
The codes have already been run through clang-format.

### CODE REVIEW

File: webapp/src/main/webapp/resources/template.soy [hunk 1]
```diff
@@ -347,4 +347,39 @@
...
+
+/**
+ * Renders a basic explanation of the stops too close notice.
+ * @param notices : List<[stopId1 : string, csvRowNumberStop1 : number, stopId2 : string, csvRowNumberStop2 : number, tripBufferMeters : number]>
+ */
+{template .stopsTooClose}
+{let $numNotices: length($notices)/}
+  <p class="warning">Warning - Stops too close!</p>
+  <p>Description: Two stops are too close with each other.</p>
+  <p><b>{$numNotices}</b> Stops too close found in:</p>
```
Comment (lilian-hunt) [inline webapp/src/main/webapp/resources/template.soy]: Same as previous, I think `<p><b>{$numNotices}</b> found:</p>` is sufficient

File: webapp/src/main/webapp/resources/template.soy [hunk 2]
```diff
@@ -347,4 +347,39 @@
...
+  <table>
+    <thead>
+      <tr>
+        <th>Stop ID 1</th>
+        <th>CSV Row Number Stop 1</th>
+        <th>Stop ID 2</th>
+        <th>CSV Row Number Stop 2</th>
+        <th>Trip Buffer Meters</th>
```
Comment (lilian-hunt) [inline webapp/src/main/webapp/resources/template.soy]: Trip Buffer in Meters

File: webapp/src/main/webapp/resources/template.soy [hunk 3]
```diff
@@ -347,4 +347,39 @@
...
+{/template} 
+
+/**
+ * Renders a basic explanation of the stops too close notice.
+ * @param notices : List<[stopId1 : string, csvRowNumberStop1 : number, stopId2 : string, csvRowNumberStop2 : number, tripBufferMeters : number]>
+ */
+{template .stopsTooClose}
+{let $numNotices: length($notices)/}
+  <p class="warning">Warning - Stops too close!</p>
+  <p>Description: Two stops are too close with each other.</p>
```
Comment (lilian-hunt) [inline webapp/src/main/webapp/resources/template.soy]: too close **to** each other

File: webapp/src/main/webapp/resources/template.soy (hunk 1 above)
Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: Sounds reasonable~ Will go to adjust them all.

File: webapp/src/main/webapp/resources/template.soy (hunk 2 above)
Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: Thank you! Fixed.

File: webapp/src/main/webapp/resources/template.soy (hunk 3 above)
Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: Thank you! Fixed.

File: webapp/src/main/webapp/resources/template.soy (hunk 1 above)
Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: Fixed. Will fix in the corresponding following PRs.

Comment (KathleenQ) [reply webapp/src/main/webapp/resources/template.soy]: All changes are in PR #46

**Review by jupi3200 — APPROVED** [review]

**Pull request merged**